driver/panda_stream.c

```diff
@@ -282,6 +290,8 @@ static void start_hardware(struct stream_open *open)
     open->isr_block_index = 0;
     open->read_block_index = 0;
     open->read_offset = 0;
+    // having a zeroed timestamp means the start event did not happen
+    open->start_ts = (const struct timespec64){0};
```

Comment:
Yes, this makes sense, and ensures that each time we arm the timestamp is reset.